modules/eco-poison-pill-operator-about.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="about-poison-pill-operator_{context}"]
+= About the Poison Pill Operator
+
+The Poison Pill Operator runs on the cluster nodes and reboots nodes that are identified as unhealthy. The Operator uses the `MachineHealthCheck` controller to detect the health of a node in the cluster. When a node is identified as unhealthy, the `MachineHealthCheck` resource creates the `PoisonPillRemediation` custom resource (CR), which triggers the Poison Pill Operator. 
+
+The Poison Pill Operator provides the following capabilities:
+
+* Minimizes downtime for stateful applications and restores compute capacity if transient failures occur.
+* Independent of any management interface, such as IPMI or an API to provision a node. 
+
+[id="understanding-poison-pill-operator-config_{context}"]
+== Understanding the Poison Pill Operator configuration
+
+The Poison Pill Operator creates the `PoisonPillConfig` CR with the name `poison-pill-config` in the Poison Pill Operator's namespace. You can edit this CR. However, you cannot create a new CR for the Poison Pill Operator.
+
+A change in the `PoisonPillConfig` CR re-creates the Poison Pill daemon set.
+
+The `PoisonPillConfig` CR resembles the following YAML file:
```

Comment:
```suggestion
The `PoisonPillConfig` definition resembles the following YAML file:
```

Reply:
> 
Retaining `PoisonPillConfig` CR per our discussion
Addressed the remaining comments. Thank again!